tRPC-Go-#2001.md

```
@@ -0,0 +1,536 @@
+- **Summary filter-key**：`session.Summary` 支持按 `FilterKey` 分层管理摘要，非空 key 表示按 event filter 分支汇总，空 key 表示全量摘要。比较时需检查 filter-key 归属、覆盖关系和版本
+- **Track 观测轨迹**：`session.TrackService.AppendTrackEvent` 存储工具执行耗时、子任务状态、异常记录等，Go 版多处已有实现（Redis、SQLite、Postgres、MySQL 均支持 Track）
+- **事件分页**：部分后端（Postgres、MySQL）支持 `WithGetSessionEventPage`，需标记 unsupported 后端
+- **TTL**：部分后端支持 `sessionTTL`，需在比较时考虑
+
+---
+
+## 五、框架设计
+
+### 5.1 目录结构
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)